src/droid.cpp

```diff
@@ -2889,7 +2889,7 @@ bool droidSensorDroidWeapon(const BASE_OBJECT *psObj, const DROID *psDroid)
 	//check vtol droid with vtol sensor
 	if (isVtolDroid(psDroid) && psDroid->asWeaps[0].nStat > 0)
 	{
-		if (psStats->type == VTOL_INTERCEPT_SENSOR || psStats->type == VTOL_CB_SENSOR || psStats->type == SUPER_SENSOR || psStats->type == RADAR_DETECTOR_SENSOR)
+		if (psStats->type == VTOL_INTERCEPT_SENSOR || psStats->type == VTOL_CB_SENSOR || psStats->type == SUPER_SENSOR /*|| psStats->type == RADAR_DETECTOR_SENSOR*/)
```

Comment:
Please do not leave behind commented-out code in the codebase.

Reply:
I don't usually leave commented out code but it's important to keep an easy to find record in case we want to actually allow assigning units to the radar detector again. Something you forgot to disable a long time ago.